build/config.json

```diff
@@ -1 +1 @@
-{"elasticsearch":{"httpAuth":"","host":"http://localhost:8080/api/catalog","index":"vue_storefront_catalog"},"cart":{"synchronize":true,"create_endpoint":"http://localhost:8080/api/cart/create?token={{token}}","updateitem_endpoint":"http://localhost:8080/api/cart/update?token={{token}}&cartId={{cartId}}","deleteitem_endpoint":"http://localhost:8080/api/cart/delete?token={{token}}&cartId={{cartId}}","pull_endpoint":"http://localhost:8080/api/cart/pull?token={{token}}&cartId={{cartId}}"},"orders":{"endpoint":"http://localhost:8080/api/order"},"users":{"endpoint":"http://localhost:8080/api/user"},"stock":{"endpoint":"http://localhost:8080/api/stock"},"images":{"baseUrl":"https://demo.vuestorefront.io/img/"},"install":{"is_local_backend":true,"backend_dir":"../vue-storefront-api"},"demomode":true,"tax":{"defaultCountry":"PL","defaultRegion":"","calculateServerSide":false},"i18n":{"defaultCountry":"US","defaultLanguage":"EN"},"mailchimp":{"endpoint":"http://localhost:8080/api/ext/mailchimp-subscribe/subscribe"},"registeredExtensions":["custom_extension","mailchimp-subscribe"]}
\ No newline at end of file
+{"elasticsearch":{"httpAuth":"","host":"localhost:8080/api/catalog","index":"vue_storefront_catalog"},"cart":{"synchronize":true,"create_endpoint":"http://localhost:8080/api/cart/create?token={{token}}","updateitem_endpoint":"http://localhost:8080/api/cart/update?token={{token}}&cartId={{cartId}}","deleteitem_endpoint":"http://localhost:8080/api/cart/delete?token={{token}}&cartId={{cartId}}","pull_endpoint":"http://localhost:8080/api/cart/pull?token={{token}}&cartId={{cartId}}"},"orders":{"endpoint":"http://localhost:8080/api/order"},"users":{"endpoint":"http://localhost:8080/api/user"},"stock":{"endpoint":"http://localhost:8080/api/stock"},"images":{"baseUrl":"https://demo.vuestorefront.io/img/"},"install":{"is_local_backend":true,"backend_dir":"../vue-storefront-api"},"demomode":true,"tax":{"defaultCountry":"PL","defaultRegion":"","calculateServerSide":false},"i18n":{"defaultCountry":"US","defaultLanguage":"EN"},"mailchimp":{"endpoint":"http://localhost:8080/api/ext/mailchimp-subscribe/subscribe"},"registeredExtensions":["custom_extension","mailchimp-subscribe"]}
```

Comment:
Please add ignore and remove this file from this PR as it is automatically generated config dump

Reply:
@pkarw Add this file to .gitignore, Is this that you mean ?

Reply:
Yes, thank you